purpleair-aqi.js

```diff
@@ -329,13 +333,13 @@ function computePM(sensorData) {
   const hum = Number.parseInt(sensorData.hum, 10);
   const dataAverage = (adj1 + adj2) / 2;
   console.log(`PM2.5 number is ${dataAverage}.`)
-  if (dataAverage < 250) { 
-  console.log(`Using EPA calculation.`)
-  return 0.52 * dataAverage - 0.085 * hum + 5.71;
-} else {
-  console.log(`Using AQANDU calculation.`)
-  return .0778 * dataAverage + 2.65 
-}
+  if (dataAverage < 250) {
```

Comment:
Sorry, my text editor ultimately changed this on both of my branches. It should cleanly merge though.